Ginger/Ginger/Variables/VariablesPage.xaml.cs

```diff
@@ -367,8 +367,16 @@ private void VariablesGrid_grdMain_MouseDoubleClick(object sender, EventArgs e)
         {
             VariableBase selectedVarb = (VariableBase)grdVariables.CurrentItem;
             selectedVarb.NameBeforeEdit = selectedVarb.Name;
-
-            VariableEditPage w = new VariableEditPage(selectedVarb, mContext, false);
+            VariableEditPage.eEditMode editMode = VariableEditPage.eEditMode.BusinessFlow;
+            if (mContext.BusinessFlow == null)
```

Comment:
in few places we are passing context as null. So please add null check for mContext.
Otherwise we might get null exception when mContext is null